ledfx/devices/__init__.py

```diff
@@ -725,9 +725,7 @@ async def add_new_device(self, device_type, device_config):
 
         # Create the device
         _LOGGER.info(
-            "Adding device of type {} with config {}".format(
-                device_type, device_config
-            )
+            f"Adding device of type {device_type} with config {device_config}"
```

Comment:
Is this just f-stringing because you can ( i am a fan of fstring ) or is something forcing this change

Reply:
Potentially pyupdater doing this.

Reply:
Either that or linting perhaps? Either way, I didn't touch this myself.